pkg/asset/installconfig/azure/validation.go

```diff
@@ -212,8 +221,11 @@ func validateInstanceTypes(client API, ic *types.InstallConfig) field.ErrorList
 				ultraSSDCapability = defaultUltraSSDCapability
 			}
 			ultraSSDEnabled := strings.EqualFold(ultraSSDCapability, "Enabled")
+			if acceleratedNetworking == "" {
+				acceleratedNetworking = string(aztypes.AcceleratedNetworkingBasic)
```

Comment:
Why do we default to basic networking for compute nodes?

Reply:
This depends on instance type. AcceleratedNetworking requires 4 vCPU. We used to use 2 vCPU instances as the nodes. Since then we have updated this to use 4vCPU for our compute nodes. We can update this to accelerated.

Reply:
We will default this to `Accelerated`.